app/ledger.js

```diff
@@ -323,7 +323,7 @@ var boot = () => {
       appActions.updateLedgerInfo({})
 
       bootP = false
-      return console.log('ledger client boot error: ' + ex.toString() + '\n' + ex.stack)
+      return console.error('ledger client boot error: ' + ex.toString() + '\n' + ex.stack)
```

Comment:
Same as above- stack trace no longer needed since it'll be included w/ console.error